runners/flink/job-server/flink_job_server.gradle

```diff
@@ -139,12 +139,9 @@ def portableValidatesRunnerTask(String name, Boolean streaming) {
       includeCategories 'org.apache.beam.sdk.testing.ValidatesRunner'
       excludeCategories 'org.apache.beam.sdk.testing.FlattenWithHeterogeneousCoders'
       excludeCategories 'org.apache.beam.sdk.testing.LargeKeys$Above100MB'
-      excludeCategories 'org.apache.beam.sdk.testing.UsesAttemptedMetrics'
```

Comment:
Not a blocker for this PR but out of curiosity, do enabling these in the Portable Spark Runner pass? It would be a good idea to enable it to if so, or report the errors so they can be fixed if not.

Reply:
I'm not sure. I can create a PR to check this, that's a topic worth investigating.

Reply:
https://github.com/apache/beam/pull/10198 it's here

Reply:
Hmm, now that I think of it, you probably wanted to check the impact of my changes on the Spark runner, right?

Reply:
@iemejia Enabling the tests on Portable Spark runner fails. I'd have to investigate further in order to   pinpoint the areas that fail